README.rst

```diff
@@ -62,17 +62,17 @@ directory.
 Releases
 ========
 
-You can find release notes at https://docs.scrapy.org/en/latest/news.html
+You can check https://docs.scrapy.org/en/latest/news.html for release notes.
```

Comment:
It comes from the original string, but since we are at it…

```suggestion
You can check https://docs.scrapy.org/en/latest/news.html for the release notes.
```

Reply:
Great! =)

Reply:
@josealberto4444 can you please commit this suggestion so that we could merge this?